client/post-editor/editor-notice/index.jsx

```diff
@@ -192,8 +193,27 @@ export class EditorNotice extends Component {
 		}
 	}
 
+	renderNoticeAction() {
+		const { onViewClick, action, link } = this.props;
+		if ( onViewClick && ! this.props.site.jetpack ) {
```

Comment:
Though highly unlikely - `this.props.site` could be undefined, perhaps we could use [isJetpackSite](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors.js#L142) here instead just to be safe.